kythe/go/platform/tools/viewindex/viewindex.go

```diff
@@ -61,6 +66,61 @@ func main() {
 	}
 
 	path := flag.Arg(0)
+	if !strings.HasSuffix(path, ".kzip") {
+		// Assume deprecated .kindex, by default
```

Comment:
does it actually have a .kindex suffix usually?  I thought so, but maybe not.

Reply:
It does, but the previous behavior took any extension.   I'm leaving that (minus ".kzip") until .kindex is removed.